packages/integrations/useFocusTrap/index.ts

```diff
@@ -60,7 +61,7 @@ export interface UseFocusTrapReturn {
  * @see https://vueuse.org/useFocusTrap
  */
 export function useFocusTrap(
-  target: MaybeElementRef,
+  target: string | MaybeComputedElementRef | (MaybeComputedElementRef | string)[] | MaybeRefOrGetter<(MaybeComputedElementRef | string)[]>,
```

Comment:
Modifying the type to `Arrayable<MaybeRefOrGetter<string> | MaybeComputedElementRef>` seems more convenient.

Reply:
I didn’t realize there was such a type, thanks for pointing it out.